alfresco-bulk-export/src/main/resources/alfresco/module/org.alfresco.extension.bulkexport/bulk-export-job-content.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Change file name from bulk-export-job-content.xml to bulk-export-job-context.xml